src/player_computer.c

```diff
@@ -1512,6 +1513,10 @@ void setup_computer_players2(void)
             skirmish_AI_type = 7;
         }
         setup_a_computer_player(i, skirmish_AI_type);
+        if ((game.flags_font & FFlg_AlexCheat) != 0)
```

Comment:
This seems outdated code too. This is compuchat now, right?

Reply:
Yes!

Reply:
So, will you change it?

Reply:
fixed